app/assets/stylesheets/components.scss

```diff
@@ -225,6 +226,11 @@
   &__download {
     margin-top: 1.5em;
   }
+  .show-toggle {
```

Comment:
This should use the BEM convention now.

Reply:
This is overriding a gov uk class. 
Have changed to 'dgu-datalinks__toggle'

Reply:
No longer required as we are now using 'button secondary' as requested below